### ISSUE TITLE
question in lrp_layers.py

### ISSUE THREAD

**Issue by S200331082** [kaifishr/PyTorchRelevancePropagation#3]
hi @kaifishr 
Thanks for your implementation. I'm trying to reimplement lrp on Resnet50, but it has a BatchNorm2D layer in the backbone, I'm a freshman in python and I don't know how to code the `RelevancePropagationBatchNorm2D` in lrp_layers.py. Can you just give me some ideas？ Thanks a lot.

**Comment by kaifishr** [comment]
As `BatchNorm2D` consists of two consecutive affine linear transformations I would try to weight the relevance scores by the weight parameters of the batch normalization layer learned during the training.

**Comment by zbb2022hust** [comment]
Hello, I also meet problems when calculating relevance via the BatchNorm1D layer. I'm not professional on math, but I'm in urgent to use this method to evaluate my FCN model in data-driven fault diagnosis task. Could you add the RelevancePropagationBatchNorm1d/2d in lrp_layers or explain more clear on how to calculate this? 
Thanks! 
Best regards.

**Comment by peacefulotter** [comment]
Bumping this as BatchNorm2D is required for ResNet models!
